src/openMVG/matching_image_collection/Cascade_Hashing_Matcher_Regions_AllInMemory.cpp

```diff
@@ -144,7 +144,7 @@ void Match
 #endif
     for (int j = 0; j < (int)indexToCompare.size(); ++j)
     {
-      const size_t J = indexToCompare[j];
+      size_t J = indexToCompare[j];
```

Comment:
if you have : 

```
const size_t J
```

using std::move on it produces this type : 
```
const size_t &&
```

but that type can't be used later in std::make_pair : 

```
openMVG/src/openMVG/matching_image_collection/Cascade_Hashing_Matcher_Regions_AllInMemory.cpp:212:63: error: 
      binding value of type 'const size_t' (aka 'const unsigned long') to reference to type 'unsigned long' drops 'const' qualifier
          map_PutativesMatches.insert( make_pair( make_pair(I,J), std::move(vec_putative_matches) ));
```